### PR TITLE
refactor: simplify tests

### PR DESCRIPTION
**Pull request by SilasPeters**
## Description

*Desribe what you have done in this pull request*

## Testability

*Describe how to test your changes*

## Checklist
- [x] Set the proper pull request name
- [x] Merged main into your branch
- [x] Added a scene to the game for your changes

### CODE REVIEW

**Review by JensSteenmetz — DISMISSED** [review]
Awesome

**Review by QuakeEye — APPROVED** [review]

File: aplib.net-demo/Assets/Scripts/PipelineTesting/Calculator.cs [hunk 1]
```diff
@@ -1,5 +1,3 @@
-using System.Collections;
-using System.Collections.Generic;
 using UnityEngine;
 
 public class Calculator : MonoBehaviour
```
[the remark this answers is not in the document]
Comment (SilasPeters) [reply aplib.net-demo/Assets/Scripts/PipelineTesting/Calculator.cs:3]: Niiiiiiiiiice

**Review by JensSteenmetz — APPROVED** [review]
LGTM!!

**Review by QuakeEye — APPROVED** [review]
image

**Pull request merged**